app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -196,6 +196,9 @@ import com.duckduckgo.downloads.api.DOWNLOAD_SNACKBAR_LENGTH
 import com.duckduckgo.downloads.api.DownloadCommand
 import com.duckduckgo.downloads.api.FileDownloader
 import com.duckduckgo.downloads.api.FileDownloader.PendingFileDownload
+import com.duckduckgo.js.messaging.api.JsMessageHandler
```

Comment:
All the changes in this file as for testing purposes, they will all be reverted before merging.